cloud_functions/triggers/trigger_climateiq_export_pipeline.py

```diff
@@ -0,0 +1,52 @@
+from google.cloud import storage, pubsub_v1
+import argparse
+
+project_id = "climateiq"
+topic_id = "climateiq-spatialize-and-export-predictions"
+bucket_name = "climateiq-predictions"
+
+
+def publish_messages(prediction_type, model_id):
```

Comment:
@waltaskew are there any signals we can use determine when the ML model has finished processing all outputs? USL would like to manually initiate the export pipeline so they'll be calling this script once they're ready.

the signal could be as simple as "check this vertex AI UI and see if the check mark is there"

Reply:
This might be a good thing for you, Prashant and I to talk about once Prashant it back next week.
My inclination would be to create one batch VertexAI job for the entire study area, which would generate predictions for every chunk in the batch. So that would give us a single vertexai job to look at.

This may have an impact on how VertexAI writes its outputs, though. It might result in all predictions being in a single file, which would affect our strategy for parallel export processing.

Reply:
SG, set up some time for us to chat on Monday

Reply:
Somewhat related, should there be some kind of run_id/batch_id attached to each VertexAI job which should be passed as a parameter to the spatialize_chunk_predictions cloud function?

Reply:
hmm what would the run_id/batch_id be used for?

Reply:
We're now passing a random ID associated with a specific batch VertexAI job